docs/01-overview/02-main-areas/application-connectivity/ac-02-application-gateway.md

```diff
@@ -1,19 +1,18 @@
 ---
 title: Application Gateway
```

Comment:
It is quite a long and detailed doc, I'm not sure if all its content should land in the Main areas section. The content is duplicated in  **Application Gateway details**.